Using TensorFlow backend.
2018-02-19 12:37:00.024923 Start.
Training mixture of experts with adaptive threshold on BirdVox-70k with PCEN input. 
Training set: unit07, unit10, unit01.
Validation set: unit02, unit03.
Test set: unit05.

h5py version: 2.6.0
keras version: 2.0.6
numpy version: 1.13.1
pandas version: 0.20.3
pescador version: 1.0.0
tensorflow version: 1.2.1

Traceback (most recent call last):
  File "../../../src/054_train-pcen-addntt-convnet.py", line 371, in <module>
    validation_steps = validation_steps)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/legacy/interfaces.py", line 87, in wrapper
    return func(*args, **kwargs)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/engine/training.py", line 1840, in fit_generator
    class_weight=class_weight)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/engine/training.py", line 1559, in train_on_batch
    check_batch_axis=True)
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/engine/training.py", line 1238, in _standardize_user_data
    exception_prefix='target')
  File "/home/vl1019/miniconda3/lib/python3.6/site-packages/keras/engine/training.py", line 128, in _standardize_input_data
    str(array.shape))
ValueError: Error when checking target: expected dense to have 3 dimensions, but got array with shape (32, 1)
